packages/swatch/src/SwatchGroup.ts

```diff
@@ -196,6 +204,18 @@ export class SwatchGroup extends SizedMixin(SpectrumElement, {
         const passThroughSwatchActions: ((swatch: Swatch) => void)[] = [];
         if (Object.keys(targetValues).length) {
             passThroughSwatchActions.push((swatch) => {
+                if (window.__swc.DEBUG) {
+                    if (
+                        'selects' in targetValues &&
+                        targetValues['selects'] !== 'multiple'
```

Comment:
Which part of this test for `mixed-value`?

Reply:
Should be there now. Silly miss!